### ISSUE TITLE
Unable to use vmware_vm_shell on latest AWX-EE

### ISSUE THREAD

**Issue by mhorvat17** [ansible/awx-ee#142]
Hi Everyone,

As of 11th October we are experiencing an issue where our VM builds using the latest EE are failing. We are able to use most of the community.vmware modules but when we try to use vmware_vm_shell it is failing saying that "Failed to import the required Python library (PyVmomi)"

Has anyone else experienced this issue?
It does appear to have lined up with the last releases of the EE image.

  "exception": "Traceback (most recent call last):\n  File \"/tmp/ansible_vmware_vm_shell_payload_ci5046fc/ansible_vmware_vm_shell_payload.zip/ansible_collections/community/vmware/plugins/module_utils/vmware.py\", line 36, in <module>\n    from pyVim import connect\nModuleNotFoundError: No module named 'pyVim'\n",

  "msg": "Failed to import the required Python library (PyVmomi) on automation-job-1278-x74pk's Python /usr/libexec/platform-python. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter",

**Comment by gdlx** [comment]
Yes I have the same error with `boto3` while trying to upgrade from v19 to v21 of AWX (and EE `0.5.0` to `latest`):
```
Failed to import the required Python library (botocore or boto3) on automation-job-19960-xj4rr's Python /usr/libexec/platform-python. Please read the module documentation and install it in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter
```
@mhorvat17 Have you been able to fin a workaround ?

**Comment by john-westcott-iv** [comment]
We're you all able to get this resolved? Does the latest EE work?
If something isn't included within the EE you typically need to roll your own. However, for dependencies on collections included the EE build process should pick them up if they are properly defined in the collections.

Please let us know if this is still an issue or if you were able to resolve.
